smartdevicelink/sdl_core
# Remove unused policy table fields

Fixes #3127

This PR is **ready** for review.

### Risk
This PR makes **no** API changes.

### Testing Plan
1. Open file [types.h](https://github.com/smartdevicelink/sdl_core/blob/master/src/components/policy/policy_external/include/policy/policy_table/types.h#L229)
2. Make sure that all fields from listed structures are used in the project

### Summary
Deleted unused fields in ApplicationParams and ModuleConfig structs.

### CLA
